container/openejb-core/src/main/java/org/apache/openejb/config/ActivationConfigPropertyOverride.java

```diff
@@ -130,6 +142,34 @@ public AppModule deploy(final AppModule appModule) throws OpenEJBException {
         return appModule;
     }
 
+    private MdbContainer getMdbContainer(final String containerId) {
+
+        final ContainerSystem containerSystem = SystemInstance.get().getComponent(ContainerSystem.class);
+
+        if (containerSystem == null) {
+            return null;
+        }
+
+        if (containerId == null || containerId.length() == 0) {
```

Comment:
shouldn't be possible, if it is then we should fail probably

You can also index (Map<String/*id*/, MdbContainer>) the container, depending the number of MDB it can be worth it